EIPS/eip-5792.md

```diff
@@ -0,0 +1,278 @@
+* The wallet MUST attempt to deliver all messages if it returns a successful response to this method, and the wallet MUST
+* NOT deliver any messages if it returns an error response.
+* The wallet MAY reject the request if the chain ID does not match the currently selected chain ID.
+* The wallet MUST send the messages on the specified chain ID.
+* The wallet MUST reject the request if the `from` address does not match the enabled account.
+* The wallet MAY reject the request if one or more messages in the bundle will fail.
+
+#### Parameters
+
+The method takes an array containing one object element, with the following keys:
+
+- `chainId`: `INTEGER` - The chain ID on which the transactions should be sent
+- `from`: `ADDRESS` - The address of the wallet that should send the messages in the bundle
+- `messages`: `ARRAY` - An array containing 1 or more message objects, where each message has the following keys:
+    - `to`: `ADDRESS` - (optional when creating new contract) The address the message is directed to.
+    - `gas`: `QUANTITY` - (optional) Integer of the gas provided for the message execution. Unused gas MAY be made
+      available to subsequent messages.
+    - `value`: `QUANTITY` - (optional) Integer of the value sent with this transaction.
+    - `data`: `DATA` - The compiled code of a contract OR the message data to include with the call, e.g. encoded
+      function signature and parameters.
+    - `optional`: `BOOLEAN` - (optional, default false) Whether the wallet should continue with subsequent messages if
+      this particular message fails
+
+##### Example
+
+```json
+[
+  {
+    "chainId": 1,
+    "from": "0xd46e8dd67c5d32be8058bb8eb970870f07244567",
+    "messages": [
+      {
+        "to": "0xd46e8dd67c5d32be8058bb8eb970870f07244567",
+        "gas": "0x76c0",
+        "value": "0x9184e72a",
+        "data": "0xd46e8dd67c5d32be8d46e8dd67c5d32be8058bb8eb970870f072445675058bb8eb970870f072445675",
+        "optional": true
+      },
+      {
+        "to": "0xd46e8dd67c5d32be8058bb8eb970870f07244567",
+        "gas": "0xdefa",
+        "value": "0x182183",
+        "data": "0xfbadbaf01"
+      }
+    ]
+  }
+]
+```
+
+#### Return value
+
+DATA, 32 Bytes - an identifier that the wallet uses to represent the bundle of messages. 
+
+* This identifier MUST persist between connections of the same Dapp and account
+* The dapp may track this identifier to check the status of the transaction in the future
+* The returned identifier value MUST be unique for the given `from` address
+* The length of the returned response MUST NOT exceed 66 characters
+* There are no other requirements on the format of the identifier
+
+##### Example
+
+The identifier may be the hash of the message bundle, e.g.:
+
+```json
+"0xe670ec64341771606e55d6b4ca35a1a6b75ee3d5145a99d05921026d1527331"
+```
+
+The identifier may be a numeric identifier represented as a string, e.g.:
+
+```json
+"0x01"
+```
```

Comment:
Above, the identifier is specified as having 32-bytes. This example only gives a single byte.

So is `0x01 == 0x0000000000000000000000000000000000000000000000000000000000000001` or is `0x01 == 0x0100000000000000000000000000000000000000000000000000000000000000`?

Reply:
Changed it to a string